### ISSUE TITLE
Node initial block download is a continually annoying pain point (regtest/dev)

### ISSUE THREAD

**Issue by rt121212121** [bitcoin-sv/block-headers-client#10]
The node starts a blockchain in what is called "Initial block download" mode. This does not send P2P notifications until a certain amount of work is accrued or something. HeaderSV appears to work inconsistently with regard to this mode.

1. If HeaderSV has existing data files, it will not sync until IBD is exited.
2. If HeaderSV is started after the blocks (insufficient to exit IBD in the first case) are broadcast it appears to sync.

For regtest development IBD is a continual nuisance in any software that has to deal with the node and it is very unlikely we can get the node to change. This should be looked into and HeaderSV should have the ability to work around it and provide a consistent and reliable experience IMO.

**Comment by rt121212121** [comment]
To illustrate 1. this image is where I started HeaderSV against an empty node and then broadcast 115 blocks (which did not exit IBD).

![image](https://user-images.githubusercontent.com/32600614/163045447-b825ca66-bd77-47ad-8187-664b1fc83946.png)

2. was observed by deleting the temp jcl directory and then starting headersv against the node with the existing 115 blocks and seeing it synced them (it was unclear if it was able to sync any additional blocks before the node exited IBD I did not look).
